addons/controls/src/ControlsPanel.tsx

```diff
@@ -18,6 +20,20 @@ export const ControlsPanel: FC = () => {
     {}
   );
 
+  useEffect(() => {
+    if (
+      Object.values(rows).some(({ control: { options = {} } = {} }) =>
+        Object.values(options).some((v) => !['boolean', 'number', 'string'].includes(typeof v))
```

Comment:
@ghengeveld after upgrading to latest beta I'm getting this error: `Uncaught TypeError: _ref$control is null`